ui/desktop/app/templates/scopes/scope/projects/targets/index.hbs

```diff
@@ -23,6 +23,22 @@
           aria-label={{t 'actions.search'}}
           {{on 'input' this.handleSearchInput}}
         />
+        <S.Generic>
+          <Dropdown
+            @name={{t 'resources.target.titles.active-sessions'}}
+            @itemOptions={{this.availableSessionOptions}}
+            @checkedItems={{this.availableSessions}}
+            @applyFilter={{fn this.applyFilter 'availableSessions'}}
+          />
+        </S.Generic>
+        <S.Generic>
```

Comment:
I think we may have said that we should hide this filter if there's only one option in the filter dropdown, I made a [ticket](https://hashicorp.atlassian.net/browse/ICU-12163) for this we can do later

Reply:
Thanks for getting a ticket created for that!